apps/docs/src/app/pages/landing/landing-page.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+import { RouterLink, RouterOutlet } from '@angular/router';
+import { ERoutes } from '../../shared/types';
+
+@Component({
+	standalone: true,
+	imports: [RouterLink, RouterOutlet],
+	selector: 'landing-page',
+	templateUrl: 'landing-page.component.html',
+})
+export class LandingPageComponent implements OnInit {
+	public routes: typeof ERoutes = ERoutes;
+	constructor() {}
+
+	ngOnInit() {}
```

Comment:
The `ngOnInit` and `constructor` can go if not used. We can add them again if needed in the future.

Do a quick check of the codebase to see if this applies to other places as well.

You can always try to enable the [no-useless-constructor](https://eslint.org/docs/latest/rules/no-useless-constructor) and [no-empty-lifecycle-method](https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin/docs/rules/no-empty-lifecycle-method.md) ESLint rules to avoid this from happening.